app/views/groups/_fields.html.erb

```diff
@@ -1,4 +1,5 @@
-<h2><%= locals[:heading] %></h2>
+<div class="col-10 p-3">
+  <h2><%= locals[:heading] %></h2>
```

Comment:
do we need to use locals keyword here 
locals[:heading] -> heading

Reply:
if don't write locals, then raise error